src/targets/services/deleteAccounts.js

```diff
@@ -0,0 +1,22 @@
+import CozyClient from 'cozy-client'
+import logger from 'cozy-logger'
+import polyfillFetch from './polyfillFetch'
+
+import deleteAccounts from 'cozy-harvest-lib/dist/services/deleteAccounts'
+
+const log = logger.namespace('deleteAccounts')
+
+polyfillFetch()
+
+const main = async () => {
+  const cozyClient = CozyClient.fromEnv()
+
+  await deleteAccounts(cozyClient, JSON.parse(process.env.COZY_COUCH_DOC))
+}
+;(async () => {
+  try {
+    await main()
+  } catch (error) {
+    log('critical', error.message)
+  }
+})()
```

Comment:
Is it the same as `main().catch(error => log('critical', error.message))`?

Reply:
As far as I know, it is the same. It looks more verbose, but since this is what was used for the `updateAccounts` service, I kept things consistent.